puzzle/plugins/gemini/gemini_plugin.py

```diff
@@ -38,9 +40,12 @@ def __init__(self, db=None, vtype='snv'):
         self.filters.can_filter_impact_severity = True
 
     def test_gemini_db(self):
-        """Check if self.db is a valid gemini database"""
+        """Check if self.db is a valid gemini database
+            
+            Raises sqlite3.DatabaseError if not a valid databse
+        """
         gq = GeminiQuery(self.db)
-        return
+        return True
```

Comment:
No this is just a safety check. If `self.db` is not a valid database it will fail.
Should probably raise an error, nice spotted!